tools/online_editor/src/index.ts

```diff
@@ -96,8 +114,41 @@ function create_open_menu(editor: EditorWidget): Menu {
             editor.project_from_url(url);
         },
     });
+    commands.addCommand("slint:add_file", {
+        label: "Add a file to your Slint project",
+        iconClass: "fa-regular fa-file",
+        mnemonic: 1,
+        execute: () => {
+            let name = prompt("Please enter the file name");
+            if (name == null) {
+                return;
+            }
+            if (!name.endsWith(".slint")) {
+                name = "${name}.slint";
+            }
+            editor.add_empty_file_to_project(name);
+        },
+    });
+    commands.addCommand("slint:store_github_token", {
+        label: "Store github access token in your browsers local storage.",
+        iconClass: "fa-brands fa-github",
+        execute: () => {
+            let action = "Please provide a github access token:";
```

Comment:
The text should also explain how to get this token.

Reply:
Perhaps like wizard dialog implemented using the HTML dialog element?